ablator/modules/metrics/stores.py

```diff
@@ -314,7 +314,7 @@ def evaluate(self) -> dict[str, float]:
         {'mean': 5.333333333333334}
         """
         if self._keys is None:
-            raise RuntimeError("PredictionStore has no predictions to evaluate.")
+            return {}
```

Comment:
Did we add a test for this change? @h21565897

Reply:
I will add a test for this.

Reply:
Already fix that. See https://github.com/fostiropoulos/ablator/pull/67